core/lexicon/en/setting.inc.php

```diff
@@ -625,6 +625,9 @@
 $_lang['setting_resource_tree_node_name'] = 'Resource Tree Node Field';
 $_lang['setting_resource_tree_node_name_desc'] = 'Specify the Resource field to use when rendering the nodes in the Resource Tree. Defaults to pagetitle, although any Resource field can be used, such as menutitle, alias, longtitle, etc.';
 
+$_lang['setting_resource_pagetitle_autocomplete'] = 'Enables autocomplete for pagetitle field';
+$_lang['setting_resource_pagetitle_autocomplete_desc'] = 'Select \'Yes\' if you want to see hints/autocomplete data based on the previous entered values for the pagetitle field of the resource.';
```

Comment:
Could you change quotes to `"` for "Yes" or rephrase the string to avoid issues with backslashed symbols?

Reply:
i take example as standart from here, here uses single quotes with backslash 
![image](https://user-images.githubusercontent.com/8276064/55463216-3a9f8680-5601-11e9-9a47-e27b7eca2e0f.png)

Reply:
I understand, but the existing code does not mean automatically that it is correct. Those string also should be fixed in the future.